composables/useSearch.ts

```diff
@@ -0,0 +1,11 @@
+import { localeIncludes } from 'locale-includes'
+
+export default function () {
+  function searchText(item: string, query: string, locale: string): boolean {
+    return localeIncludes(item.toLowerCase(), query.toLowerCase(), { locales: locale, sensitivity: 'base' }) as boolean
```

Comment:
`toLowerCase` is not a locale sensitive function. Just let the `sensitivity: 'base'` do this job.

Reply:
As is just a function from an import. Why not just use it directly without adding a composable ?